test/test_gpu_view.cpp

```diff
@@ -1191,12 +1191,7 @@ TEST_F(GpuViewTest, FusionReshapeVectorize) {
 
   auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
 
-  // This test allocates about 1GB of memory, so in order to avoid an OOM during
-  // this test, we manually clear the allocator after it's reached a certain
-  // threshold.
-  maybeClearAllocator();
```

Comment:
This workaround, introduced in #1382, is no longer necessary. If >4GiB are allocated in the PyTorch allocator at the beginning of the test, it will be cleared anyway. This PR reduces the size required by a factor of 16, so even if we are at 4GiB, this test should fit easily.